File: app/example/module/ExampleModule.java
package app.example.module;

import app.example.service.ExampleStorageService;
import app.example.service.FrontendService;
import app.example.storage.example_app.ExampleStorageServiceGrpc;
import dev.logos.app.AppModule;
import dev.logos.app.register.registerModule;

@registerModule
public class ExampleModule extends AppModule {
    @Override
    protected void configure() {
        services(FrontendService.class, ExampleStorageService.class);
        clients(ExampleStorageServiceGrpc::newFutureStub);
    }
}

File: app/example/web/module/example-module.ts
import {FeedServicePromiseClient} from "app/example/proto/feed_grpc_web_pb.js";
import {AppModule, registerModule} from "@logos/web/dev/logos/web/module/app-module";

// import {ExampleApp} from "../components/example-app";


@registerModule
export class ExampleModule extends AppModule {
    override configure() {
        this.addClient(FeedServicePromiseClient);
        //this.components(ExampleApp);
    }
}
